libcontainer/cgroups/devices/systemd_test.go

```diff
@@ -116,18 +117,23 @@ func TestPodSkipDevicesUpdate(t *testing.T) {
 	}
 }
 
+func skipOnCentOS7(t *testing.T) {
```

Comment:
Can we have sync.Once?

Reply:
This is test so I did not bother to, but sure, why not. Please see the second commit.